Description: Remove Windows only deps
Last-Update: 2022-09-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,6 +26,3 @@
 
 [dependencies.thiserror]
 version = "1.0"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["winuser", "handleapi", "fileapi", "namedpipeapi", "processthreadsapi", "winsock2", "processenv"]
